.github/workflows/playwright.yml

```diff
@@ -0,0 +1,73 @@
+# Workflow that runs Playwright functional tests once per day against dev infra.
+
+name: Playwright Tests
+run-name: Playwright Tests for ${{ github.sha }}
+env:
+  SLACK_CHANNEL_ID: CBX0KH5GA # #www-notify in MoCo Slack
+  SLACK_BOT_TOKEN: ${{secrets.SLACK_BOT_TOKEN_FOR_MEAO_NOTIFICATIONS_APP}}
+on:
+  schedule:
+    - cron: "0 15 * * *" # 3pm daily
+  workflow_dispatch:
+    inputs:
+      mozorg_service_hostname:
+        description: The root URL of the Mozorg service to run tests against. eg "https://www.mozilla.org/"
+        required: true
+
+jobs:
+  notify-of-test-run-start:
+    if: github.repository == "mozilla/bedrock"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Notify via Slack that tests are starting
+        uses: ./.github/actions/slack
+        with:
+          env_name: test
+          label: "Playwright Tests [${{ github.sha }}]"
+          status: info
+          channel_id: ${{ env.SLACK_CHANNEL_ID }}
+          slack_bot_token: ${{ env.SLACK_BOT_TOKEN }}
+          ref: ${{ github.sha }}
+          message: "Playwright Tests started"
+
+  playwright-tests:
+    runs-on: ubuntu-latest
+    needs: notify-of-test-run-start
+    env:
+      PLAYWRIGHT_BASE_URL: ${{ github.event.inputs.mozorg_service_hostname || "https://dev.bedrock.nonprod.webservices.mozgcp.net/" }} # Mozorg base URL
+      CI_JOB_ID: ${{ github.run_id }}
+      DRIVER: ""
+      LABEL: playwright-tests
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+    - name: Install dependencies
+      run: cd tests/playwright && npm ci && npm run install-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v4
+      if: ${{ !cancelled() }}
+      with:
+        name: playwright-report
```

Comment:
May be worth putting the hash here to avoid filename clashes or overwriting. IIRC upload-artifact@v4 prefers unique names